### PR TITLE
[NumberProgressBar] to Nuget.

### PR DESCRIPTION
**Pull request by adlair**

### CODE REVIEW

File: Android/NumberProgressBar/component/component.yaml [hunk 1]
```diff
@@ -21,3 +21,17 @@ libraries:
...
+    removeProjects:
+        - NumberProgressBar
+    installNuGets:
+      - project: NumberProgressBarSample
+        packages:
+          - Xamarin.NumberProgressBar
+
+is_shell: true
+packages:
+  ios-unified: Xamarin.NumberProgressBar, Version=1.2.0.1
```
Comment (newky2k) [inline Android/NumberProgressBar/component/component.yaml]: This should be android not ios

File: Android/NumberProgressBar/nuget/Xamarin.NumberProgressBar.nuspec [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>A slim progress indicator with a numeric percentage alongside the progress bar.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=875036</projectUrl>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=875040</licenseUrl>
+  </metadata>
+  <files>
+    <file src="output/NumberProgressBar.dll" target="lib/Xamarin.MonoAndroid23" />
```
Comment (newky2k) [inline Android/NumberProgressBar/nuget/Xamarin.NumberProgressBar.nuspec]: this should be MonoAndroid not Xamarin.MonoAndroid23

**Review by newky2k — CHANGES_REQUESTED** [review]
The sample in the component does not build, even once the library target has been fixed in the nuspec file

File: Android/NumberProgressBar/nuget/Xamarin.NumberProgressBar.nuspec (hunk 2 above)
Comment (adlair) [reply Android/NumberProgressBar/nuget/Xamarin.NumberProgressBar.nuspec]: Issues fixed.

**Review by newky2k — APPROVED** [review]

**Pull request merged**